CWE/events/sri_lanka.txt

```diff
@@ -18,16 +18,16 @@ country_event = {
 		
 		is_possible_vassal = SRL
 		
-		#Dominion of India
-		HND = { 
-			exists = yes 
-			is_vassal = no 
-			OR = { 
-				government = hms_government 
-				government = hms_government1 
-			} 
-		}
-	}
+		#Dominion of India			
+#		HND = { 
+#			exists = yes 
+#			is_vassal = no 
+#			OR = { 
+#				government = hms_government 
+#				government = hms_government1 
+#			} 
+#		}
+#	}
```

Comment:
An extra bracket was commented out

Reply:
Fixed that and altered the other things.